CSCapstone/urls.py

```diff
@@ -10,10 +10,11 @@
 
 urlpatterns = [
 	url(r'^admin/', admin.site.urls),
+	url(r"^search/", include("watson.urls", namespace="watson")),	
 	url(r'^', include('AuthenticationApp.urls')),
+	url(r'^', include('UniversitiesApp.urls')),
 	url(r'^', include('CSCapstoneApp.urls')),
 	url(r'^projects/', include('ProjectsApp.urls')),
     url(r'^', include('CompaniesApp.urls')),
     url(r'^', include('GroupsApp.urls')),
-	url(r'^university/', include('UniversitiesApp.urls')),
```

Comment:
Lets create a new file `UniversitiesApp/urls_courses.py` (just like urls.py) and have it contain the courses urls. Then add an additional line to `CSCapstone/urls.py` for `url(r'^course/', include('UniversitiesApp.urls_courses')),`